### PR TITLE
Updating documentation in example notebooks

### LINKED ISSUE #51: examples
Hi all,

Would it be possible to include at least ~1-2 examples where all the input datasets are publicly available? 

More precisely, after installing `plasticparcels`, I can't reproduce any of the examples, as e.g. [example_Italy_coast](https://plastic.oceanparcels.org/en/latest/examples/example_Italy_coast.html)  requires the model settings:

```
# Load the model settings
settings_file = 'docs/examples/example_Italy_coast_settings.json'
settings = pp.utils.load_settings(settings_file)
```
which are not part of the path. 

Also, looking at those example configuration files [in the repo](https://github.com/OceanParcels/plasticparcels/tree/main/docs/examples), they all rely on private storage `/storage/shared/oceanparcels/input_data/`.

openjournals/joss-reviews#7094

### PR DESCRIPTION
**Pull request by michaeldenes**
Updating all documentation in example notebooks that require data.

With #55 , this is to close #51

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,21 @@ The tool is based on the [`Parcels`](https://oceanparcels.org/) computational La
...
+### Required data
+
+`plasticparcels` has been developed for use with data from the Copernicus Marine Service, and requires the following data to run:
+
+* Hydrodynamic model data: [MOI GLO12 (psy4v3r1)](https://www.mercator-ocean.eu/en/solutions-expertise/accessing-digital-data/product-details/?offer=4217979b-2662-329a-907c-602fdc69c3a3&system=d35404e4-40d3-59d6-3608-581c9495d86a)
+* Biogeochemical model data: [MOI BIO4 (biomer4v2r1)](https://www.mercator-ocean.eu/en/solutions-expertise/accessing-digital-data/product-details/?offer=8d0c01f3-81c7-0a59-0d06-602fdf63c5b6&system=dc40b324-7de7-0732-880b-5d9dcf7d344a)
+* Wave data: [ECMWF ERA5 Wave](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels) (specifically, the variables `mean_wave_period`, `peak_wave_period`, `u_component_stokes_drift`, and `v_component_stokes_drift`.)
+* Wind data: [ECMWF ERA5 Wind](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels) (specifically, the variables `10m_u_component_of_wind` and `10m_v_component_of_wind`)
+
+For the wind and wave data, we recommend using the [CDS API](https://cds.climate.copernicus.eu/api-how-to).
```
Comment (erikvansebille) [inline README.md:25]:
```suggestion
For downloading the wind and wave data, we recommend using the [CDS API](https://cds.climate.copernicus.eu/api-how-to).
```

File: README.md [hunk 2]
```diff
@@ -13,6 +13,21 @@ The tool is based on the [`Parcels`](https://oceanparcels.org/) computational La
...
+
+To run most of the examples, you will need to update the data directories in settings `.json` files.
```
Comment (erikvansebille) [inline README.md:27]:
```suggestion
To run most of the examples, you will need to update the data directories in the relevant settings `.json` file.
```

File: docs/examples/example_Italy_coast.ipynb
Comment (erikvansebille) [inline docs/examples/example_Italy_coast.ipynb:1]: The link does not seem to be clickable in my version of the red build: https://plasticparcels--56.org.readthedocs.build/en/56/examples/example_Italy_coast.html
And same for the other notes

Comment (michaeldenes) [reply docs/examples/example_Italy_coast.ipynb:1]: Good pickup. It looks related to this issue with nbsphinx https://github.com/spatialaudio/nbsphinx/issues/468
By placing a \n in between, we can use markdown hyperlinking which should resolve this

Comment (michaeldenes) [reply docs/examples/example_Italy_coast.ipynb:1]: 50f48cf looks to have fixed this

**Comment by michaeldenes** [comment]
@erikvansebille are there any other changes for the PR, or happy to merge?

**Review by erikvansebille — APPROVED** [review]

**Comment by erikvansebille** [comment]
All good; happy for you to merge!

**Pull request merged**